src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -442,6 +442,14 @@ paths:
         required: true
         schema:
           type: string
+      
+      - name: format
+        in: path
+        description: "The service returns technical information about the requested image in JSON format. XML is also supported as it was used in version one of the IIIF API"
+        required: true
+        schema:
+          type: string
+          enum: ["json", "xml"]
 
       x-streamingOutput: true
```

Comment:
Streaming output is for "large" results: Think binaries or hundreds of records. It should be removed here